### PR TITLE
54 convert project proposal.md to latex

### PR DESCRIPTION
**Pull request by kkmp-dk**

### CODE REVIEW

File: project_proposal/tex/front_page.tex [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
+% Title
+{ \LARGE \bfseries Bierproductie \\[0.4cm]}
+A management system for brewing machines
+\begin{figure}[H]
+\centering 
+\includegraphics[scale=0.5]{images/system_drawing.png}
+\label{figure:bierproductie_system}
+\end{figure}
+
+Softwareteknologi\\
```
Comment (Kechr) [inline project_proposal/tex/front_page.tex]: Found the proper English name: Bachelor of Engineering, Software Technology

File: project_proposal/tex/signatures.tex [hunk 2]
```diff
@@ -1,104 +1,104 @@
...
-\end{footnotesize}
\ No newline at end of file
+\begin{tabular}{@{}l l} 
+\textbf{Title:} & Bierproductie \\
+& \\
+\textbf{Institution:} & University of Southern Denmark \\
+& The Faculty of Engineering, The Mærsk Mc-Kinney Møller Institute \\
+& Campusvej 55, 5230 Odense M \\
+& \\
+\textbf{Education:} & Bachelor of Engineering in Software Technology\\
```
Comment (Kechr) [inline project_proposal/tex/signatures.tex]: Replace 'in' with a comma,

File: project_proposal/tex/mainpage.tex [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+\centering 
+\includegraphics[scale=0.5]{images/system_drawing.png}
+\label{figure:system_drawing}
+\caption{System drawing} 
+\end{figure}
+
+Our distributed system consists of five sub-systems, where we are writing three
+of them.
+
+\section{OPC-UA Client}
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex]: Needs to be a subsection

File: project_proposal/tex/mainpage.tex [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+The OPC-UA client is responsible for communicating with the brewing machine via
+the OPC-UA server. This is needed in order to send and receive data via
+commands.
+
+\section{Website}
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex]: Needs to be a subsection

File: project_proposal/tex/mainpage.tex [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+The website user interface is where the client will be able to interface with
+our system.
+
+\section{REST API}
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex]: Needs to be a subsection

File: project_proposal/tex/mainpage.tex [hunk 6]
```diff
@@ -0,0 +1,376 @@
...
+    \item this Batch ID
+    \item Product type
+    \item Amount of products (total, defect and acceptable)
+    \item Amount of time used in the different states
+    \item Logging of temperature over the production time
+    \item Logging of timidity over the production time
+\end{itemize}
+
+The MES/SCADA system must be able to monitor the production and display live relevant data from the machine.
+The documentation of the system must contain an illustration that defines the different components in the setup, in relation to the ISA88^1213 Part 1 Physical Hierarchy model. The system must have a visualization that can be accessed and used to display the production data. The system must be able to collect the necessary data from the machine and calculate the OEE^131516 of the machine. The OEE must be available to be displayed by the system. The system must be able to estimate the error function associated with the different products.
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex]: This line is broken. There needs to be a backslash at '^', at ISA88^1213.

File: project_proposal/tex/mainpage.tex [hunk 7]
```diff
@@ -0,0 +1,376 @@
...
+    \end{tabularx}
+    \caption{List of requirements} 
+    \label{table:requirements}
+\end{table} 
+
+\subsection{Prioritization of requirements}
+The group has decided that at the current state, every requirement is a must have.
+A further prioritization of requirements will occur, when we split the requirements into issues.
+
+\subsection{Use Case Diagram}
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex:124]: The diagram is not under this section but moved down to the page under. Is it fixable?

File: project_proposal/tex/mainpage.tex [hunk 8]
```diff
@@ -0,0 +1,376 @@
...
+In this it is shown when the different milestones must be completed to succeed
+with the project. As mentioned above, sprints with a duration of two weeks will
+be used throughout this project. Each sprint consists of issues from the product
+backlog and is chosen at the beginning of each sprint, as de-scribed in section
+Methods. To keep track of these sprints, a burn down chart will be made, as seen
+below.
+
+\begin{figure}[H]
+\centering 
+\includegraphics[scale=0.5]{images/sprint.png}
```
Comment (Kechr) [inline project_proposal/tex/mainpage.tex]: This could be scale=0.3

File: project_proposal/tex/signatures.tex [hunk 9]
```diff
@@ -1,104 +1,104 @@
...
+
+Simon Jørgensen, sijo819@student.sdu.dk\\
+% -end- -end- -end -end -end- -end- -end -end -end- -end- -end -end -end-
+
+%Bottom of page
+%\vfill
+
+\begin{tabular}{@{}l l}
+Pages:      & 10 \\
+Appendix:   & 1
```
Comment (Kechr) [inline project_proposal/tex/signatures.tex]: What appendix is this?

**Review by Kechr — CHANGES_REQUESTED** [review]

File: project_proposal/tex/signatures.tex (hunk 9 above)
Comment (kkmp-dk) [reply project_proposal/tex/signatures.tex]: Ohh we just have to write 0 for now

File: project_proposal/tex/front_page.tex (hunk 1 above)
Comment (kkmp-dk) [reply project_proposal/tex/front_page.tex]: fix next commit

File: project_proposal/tex/signatures.tex (hunk 2 above)
Comment (kkmp-dk) [reply project_proposal/tex/signatures.tex]: fix in next commit

File: project_proposal/tex/mainpage.tex (hunk 6 above)
Comment (kkmp-dk) [reply project_proposal/tex/mainpage.tex]: I am not sure I escaped it, but that just put the next character like this 
image

then i did this instead ISA88\^{}1213
image

so maybe it is resolved you let me know?

File: project_proposal/tex/mainpage.tex (hunk 7 above)
Comment (kkmp-dk) [reply project_proposal/tex/mainpage.tex:124]: this should be fixed in the new version

File: project_proposal/tex/mainpage.tex (hunk 6 above)
Comment (kkmp-dk) [reply project_proposal/tex/mainpage.tex]: @Kechr

Comment (Kechr) [reply project_proposal/tex/mainpage.tex]: Looks good.

File: project_proposal/tex/mainpage.tex (hunk 3 above)
Comment (Kechr) [reply project_proposal/tex/mainpage.tex]: @kepet19

**Review by Kechr — CHANGES_REQUESTED** [review]
The \section{OPC-UA Client} needs to be \subsection{OPC-UA Client}

**Review by Kechr — APPROVED** [review]
Looks alright. Formatting is still a bit rough some places, but it's fine.

**Pull request merged**